solidity/test/DepositFundingTest.js

```diff
@@ -66,194 +60,6 @@ describe("DepositFunding", async function() {
     await restoreSnapshot()
   })
 
-  describe("createNewDeposit", async () => {
```

Comment:
Can you clarify why this whole test killed instead of just the supply-cap specific ones

Reply:
:facepalm: